### PR TITLE
feat(ES-33): implement the Ansible Plugin

### PR DESCRIPTION
**Pull request by eliasmueller**

### CODE REVIEW

**Comment by heidrifx** [comment]
Why is this not reviewed (or at least some reviewers assigned to it)? What blocks this PR from being processed and merged?

File: src/main/java/ust/tad/ansiblempsplugin/models/tadm/Property.java [hunk 1]
```diff
@@ -4,156 +4,181 @@
...
+
+  private PropertyType type;
+
+  private boolean required;
+
+  private Object value;
+
+  private Confidence confidence = Confidence.CONFIRMED;
+
+  private static final String INVALIDPROPERTYVALUEEXCEPTIONMESSAGE =
```
Comment (heidrifx) [inline src/main/java/ust/tad/ansiblempsplugin/models/tadm/Property.java:17]: The name is really long and confusing. I get why it's called like that but wouldn't `EXCEPTIONTEMPLATE` also work? That way the name is shorter and still carries the same meaning.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,10 +2,10 @@ plugin.technology = ansible
 # [STATIC, DYNAMIC]
 plugin.analysis-type = STATIC
 
-server.port = 8084
+server.port = 8087
```
Comment (heidrifx) [inline src/main/resources/application.properties:5]: Is the port mentioned in the README.md?

File: src/test/java/ust/tad/ansiblempsplugin/analysis/SerializeXMLTest.java [hunk 3]
```diff
@@ -1,59 +1,51 @@
...
 
-    @Value("${mps.inputModel.path}")
-    private String mpsInputPath;
+  @Value("${mps.inputModel.path}")
+  private String mpsInputPath;
 
-    @Test
-    public void serializeAnsibleToXML() throws JsonProcessingException {
-        AnsibleDeploymentModel modelToSerialize = createDummyModel();
+  // TODO FIX TESTS
```
Comment (heidrifx) [inline src/test/java/ust/tad/ansiblempsplugin/analysis/SerializeXMLTest.java:12]: This should be processed before the merge!

**Review by heidrifx — CHANGES_REQUESTED** [review]
lgtm, but some comments should be addressed before we can merge.

Also, one remark: I saw `Confidence.CONFIRMED` everywhere. Is that part of the analysis task? Was this discussed with Marcel?

**Comment by heidrifx** [comment]
Also, please change the title of the PR. This isn't a draft anymore.

**Comment by eliasmueller** [comment]
> lgtm, but some comments should be addressed before we can merge.
> 
> Also, one remark: I saw `Confidence.CONFIRMED` everywhere. Is that part of the analysis task? Was this discussed with Marcel?

Yeah the thing is, that we are pretty confident, as we use a real parser and don't parse by hand so this is fine, and discussed with Marcel

File: src/main/resources/application.properties (hunk 2 above)
Comment (eliasmueller) [reply src/main/resources/application.properties:5]: does it need to be? :D

File: src/main/java/ust/tad/ansiblempsplugin/models/tadm/Property.java (hunk 1 above)
Comment (eliasmueller) [reply src/main/java/ust/tad/ansiblempsplugin/models/tadm/Property.java:17]: idk this is copied from Marcel, I would like to keep it consistent...

File: src/test/java/ust/tad/ansiblempsplugin/analysis/SerializeXMLTest.java (hunk 3 above)
Comment (eliasmueller) [reply src/test/java/ust/tad/ansiblempsplugin/analysis/SerializeXMLTest.java]: fixed

File: src/main/resources/application.properties (hunk 2 above)
Comment (heidrifx) [reply src/main/resources/application.properties:5]: not really was just wondering :D

**Review by heidrifx — APPROVED** [review]
lgtm :D

**Pull request merged**